src/layout/_template.spec.js

```diff
@@ -108,6 +108,186 @@ const FULL_EXAMPLE = `
                 "text": 'Menu',
                 "ariaLabel": 'Toggle main menu'
             },
+            "menuLinks": {
```

Comment:
Just adding these here are not going to change the html in the snapshot and therefore isn't testing them. Setting these only change the header when we use the basic variant. So we will have to add another snapshot test